internal/testutil/test_hcls.go

```diff
@@ -1448,6 +1454,36 @@ func RHclBgpSession() RHclBgpSessionResult {
 	}
 }
 
+// packetfabric_cloud_router_quick_connect
+func RHclCloudRouterQuickConnect() RHclCloudRouterQuickConnectResult {
+
+	connectionAwsResult := RHclCloudRouterConnectionAws()
```

Comment:
I think you will need a BGP session to setup quick connect so you should use `RHclBgpSession()` instead of `RHclCloudRouterConnectionAws()`